### ISSUE TITLE
pred_epoch

### ISSUE THREAD

**Issue by MZ82020** [Khrylx/AgentFormer#14]
Hello @Khrylx , thank you for your great work. 

I didn't understand what is pred_epoch in cfg files for each dataset. Why it is different from a dataset to another in cfg files? 

Thanks

**Comment by Khrylx** [comment]
Hello,

pred_epoch is used to load the checkpoint from the pre-trained model before DLow, so it can vary for different cfgs based on the performance of the pre-trained model.

**Comment by MZ82020** [comment]
Hello, 

Thank you. 
How can I choose it please?

for the first training, it is surely that the more we advance for the times, the more the model will improve! so surely pred-epoch is higher is better right?

Thanks 


----- Mail d'origine -----
De: Ye Yuan ***@***.***>
À: Khrylx/AgentFormer ***@***.***>
Cc: MZ82020 ***@***.***>, Author ***@***.***>
Envoyé: Thu, 02 Dec 2021 17:34:00 +0100 (CET)
Objet: Re: [Khrylx/AgentFormer] pred_epoch (Issue #14)

Hello,

pred_epoch is used to load the checkpoint from the pre-trained model before DLow, so it can vary for different cfgs based on the performance of the pre-trained model.

-- 
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub:
https://github.com/Khrylx/AgentFormer/issues/14#issuecomment-984793976
-- 
Bien cordialement,
Mayssa ZAIER

**Comment by Khrylx** [comment]
That's usually the case. You can also choose it based on the accuracy of the validation set.

I'm closing this issue for now, but feel free to reopen it if you still need more clarification.
